### PR TITLE
bump version to 0.7.2

### PR DESCRIPTION
**Pull request by R-Lawton**
## Overview

Bumping version to 0.7.2

### CODE REVIEW

File: deploy/olm-catalog/cloud-resources/0.7.2/cloud-resources.v0.7.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: cloud-resource-operator
+                - name: TAG_KEY_PREFIX
+                  value: integreatly.org/
+                image: quay.io/integreatly/cloud-resource-operator:0.7.2
```
Comment (ciaranRoche) [inline deploy/olm-catalog/cloud-resources/0.7.2/cloud-resources.v0.7.2.clusterserviceversion.yaml]: this should be `quay.io/integreatly/cloud-resource-operator:v0.7.2`

File: deploy/operator.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: cloud-resource-operator
       containers:
         - name: cloud-resource-operator
-          image: quay.io/integreatly/cloud-resource-operator:v0.7.1
+          image: quay.io/integreatly/cloud-resource-operator:0.7.2
```
Comment (ciaranRoche) [inline deploy/operator.yaml]: This should also be `quay.io/integreatly/cloud-resource-operator:v0.7.2`

This is something that they want for the intly operator.

File: deploy/olm-catalog/cloud-resources/0.7.2/cloud-resources.v0.7.2.clusterserviceversion.yaml (hunk 1 above)
Comment (R-Lawton) [reply deploy/olm-catalog/cloud-resources/0.7.2/cloud-resources.v0.7.2.clusterserviceversion.yaml]: Sorry thats added now

File: deploy/operator.yaml (hunk 2 above)
Comment (R-Lawton) [reply deploy/operator.yaml]: Sorry thats added now

**Pull request merged**